features/data/102.ld.working_with_context/response102-04.excludes

```diff
@@ -0,0 +1,2 @@
+root[0]['@context']
+root[1]['@context']
```

Comment:
Why @context is an exclusion?

Reply:
@context differs between tutorial en the one produced by Context Broker -- It is reported as a discrepancy between the tutorial and the Broker output.

Reply:
LGTM